lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -10,25 +10,22 @@
 
 #   yarn devtools
 
-# with a custom devtools front_end location:
+# Download DevTools as follows (to ~/src/devtools)
+# https://chromium.googlesource.com/devtools/devtools-frontend/+/HEAD/docs/workflows.md#standalone-checkout
+
+# or use a custom devtools front_end location:
 #   yarn devtools node_modules/temp-devtoolsfrontend/
 
-chromium_dir="$HOME/chromium/src"
+dt_dir="$HOME/src/devtools/devtools-frontend"
 check="\033[96m ✓\033[39m"
 
-if [[ -n "$1" ]]; then
-  dt_dir="$1"
```

Comment:
seems a bit prescriptive, no?

Reply:
huh?

Reply:
to force the devtools checkout to be in your home directory under `src`. wasn't this giving the caller the option to pass where they have a devtools checkout?

Reply:
ah, wasn't my intention.